composer.json

```diff
@@ -78,11 +78,9 @@
                 "Importing configuration on site without password_policy activated": "https://www.drupal.org/files/issues/2019-05-20/password_policy-config_import_field_error-2771129-80.patch"
             },
             "drupal/core": {
-                "Allow updating modules with new service dependencies and Switch to a memory backend for running the database updates": "https://www.drupal.org/files/issues/2019-07-03/3055443-19-2863986-111-combined.patch"
-            },
-            "drupal/simple_sitemap": {
-                "Remove setting": "https://www.drupal.org/files/issues/2019-06-25/3063870-3.patch",
-                "Concurrent update of media module and simple_sitemap 2.x to 3.x fails": "https://www.drupal.org/files/issues/2019-06-24/3062747-8.patch"
+                "Allow updating modules with new service dependencies": "https://www.drupal.org/files/issues/2019-07-07/2863986-4-114.patch",
+                "Switch to a memory backend for running the database updates": "https://www.drupal.org/files/issues/2019-07-22/3055443-2-37.patch",
+                "Stop invoking pre-save methods in EntityStorageInterface::restore()": "https://www.drupal.org/files/issues/2019-07-23/3066439-2-58.patch"
```

Comment:
I think we should include this patch because we are shipping a paragraphs field on the tags and channel vocabulary and that can cause performance problems when switching to D8.7.